warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1013,7 +1030,6 @@ public IteratorOptions describeOptions() {
         options.put(QUERY, "The JEXL query to evaluate documents against");
         options.put(QUERY_ID, "The UUID of the query");
         options.put(TYPE_METADATA, "A mapping of field name to a set of DataType class names");
-        options.put(METADATA_TABLE_NAME, "The name of the metadata table");
```

Comment:
In describeOptions, this:
options.put(METADATA_TABLE_NAME, "The name of the metadata table");
is the correct line to leave in and should remain
options.put(METADATA_TABLE_NAME, this.metadataTableName);
below is not descriptive and should be deleted